### DISCUSSION TITLE
Atlas Glow Wallpaper #1

### DISCUSSION THREAD

**Discussion by LShenis** [Atlas-OS/Atlas#222] [Community Artwork]
![AtlasOS-Wallpaper-4K](https://user-images.githubusercontent.com/110496682/182470928-a352ebba-9dde-453d-a61c-f7fc2308754a.png)

**Comment by TheAnnity** [comment]
How to add wallpapers on atlas os tho?

**Reply by Zusier** [reply]
right click the image (after you downloaded it), set as desktop image.
